ruby/vampires.rb

```diff
@@ -0,0 +1,52 @@
+puts "How many employees are you processing?"
+num_employees = gets.chomp.to_i
+
+num_employees.times do
```

Comment:
Great job with using the times loop, if you ever needed to know what part of the iteration you are at you can use the optional block per the docs - https://ruby-doc.org/core-2.2.0/Integer.html#method-i-times